src/utils/encryption.js

```diff
@@ -0,0 +1,18 @@
+import crypto from 'crypto';
```

Comment:
Why not use bcrypt instead of native crypto? It will be a little slower but much more secure.

Reply:
Bcrypt has no decrypt functionality

Reply:
oh yeah nvm. Bcrypt hashes it, does not encrypt it.